core/src/main/resources/struts-excluded-classes.xml

```diff
@@ -37,7 +37,9 @@
                 java.lang.ProcessBuilder,
                 java.lang.Thread,
                 sun.misc.Unsafe,
-                com.opensymphony.xwork2.ActionContext"/>
+                com.opensymphony.xwork2.ActionContext,
+                com.opensymphony.xwork2.ognl.SecurityMemberAccess,
```

Comment:
Latter 2 already excluded by `struts.excludedPackageNames`

Reply:
Thanks, addressed in #729